example/README.md

```diff
@@ -0,0 +1,36 @@
+# example
+
```

Comment:
I would also add a very short description about this instruction. Something like this:
This repository contains definition docker-compose environment which makes it possible to use Jupyter Notebook using the Flink JDBC Elastic Catalog. ...

Reply:
done